src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(String color, int radius) {
+        setColor(color);
+        this.radius = radius;
+    }
+
+    public Circle(String color, int radius, String name) {
+        setName(name);
+        setColor(color);
+        this.radius = radius;

```

Comment:
This constructor is an overload of the first one with an additional parameter `name`. Consider using constructor chaining to avoid code duplication.